Automattic/jetpack
# BlogrollItem: update icon

This PR updates the icon for the blogroll item block



Fixes #
https://github.com/Automattic/wp-calypso/issues/82394

## Proposed changes:

* Add new icon

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the editor
* Check the icon for the blogroll item in jetpack site and simple site
